app/src/lib/ui/base/MainAccordion.svelte

```diff
@@ -20,14 +21,20 @@
 <li>
 	<section data-dsfr-accordion class="fr-accordion">
 		<h2 class="fr-accordion__title ">
-			<button
-				class="fr-accordion__btn !py-6 !pl-0"
-				aria-expanded={expanded}
-				aria-controls={accordionId}
-				><span class="text-france-blue text-2xl font-bold">{title}</span></button
-			>
+			{#if isFixed}
+				<div class="fr-accordion__btn !py-6 !pl-0 after:content-none" aria-controls={accordionId}>
+					<span class="text-france-blue text-2xl font-bold">{title}</span>
+				</div>
+			{:else}
+				<button
+					class="fr-accordion__btn !py-6 !pl-0"
+					aria-expanded={expanded}
+					aria-controls={accordionId}
+					><span class="text-france-blue text-2xl font-bold">{title}</span></button
+				>
+			{/if}
 		</h2>
-		<div class="fr-collapse" id={accordionId}>
+		<div class:fr-collapse={!isFixed} id={accordionId}>
```

Comment:
On pourrait peut etre rajouter un petit liseret gris entre chaque partie ? 
https://tailwindcss.com/docs/divide-width#add-borders-between-stacked-children
```suggestion
		<div class:fr-collapse={!isFixed} id={accordionId} class="divide-y" > 
```